docs/OONIProbeLegacyCompatibility.md

```diff
@@ -0,0 +1,118 @@
+# Command line flags
+
+`-h, --help` Display help and exit
+Supported: ✅
+probe-cli equivalent: `-h, --help`
+
+`-n, --no-collector` Disable writing to collector
+Supported: ❌
+Priority: high
+
+`-N, --no-njson` Disable writing to disk
+Supported: ❌
+Priority: low
+
+`-g, --no-geoip` Disable geoip lookup on start.
+Supported: ❌
+Priority: low
+
+`-s, --list` List the currently installed ooniprobe nettests
+Supported: ❌
+Priority: low
+
+`-w, --web-ui` Start the web UI
+Supported: ❌
+Priority: wontfix, we have no web UI in probe-cli
+
+`-z, --initialize` Initialize ooniprobe to begin running it
+Supported: ✅
+probe-cli equivalent: `ooniprobe onboard`
+
+`-o, --reportfile PATH_TO_FILE` Specify the report file name to write to.
+Supported: ❌
+Priority: medium
+
+`-i, --testdeck PATH_TO_DECK` Specify as input a test deck: a yaml file containing the tests to run and their arguments.
+Supported: ❌
+Priority: wontfix, we have no deck support
+
+`-c, --collector COLLECTOR_ADDRESS` Specify the address of the collector for test results. In most cases a user will prefer to specify a bouncer over this.
+Supported: partially
+probe-cli equivalent: edit ooniprobe.conf to specify the collector address in the options
+
+`-b, --bouncer BOUNCER_ADDRESS` Specify the bouncer used to obtain the address of the collector and test helpers.
+Supported: partially
+probe-cli equivalent: edit ooniprobe.conf to specify the bouncer address in the options
+
+`-l, --logfile PATH_TO_LOGFILE` Write to this logs to this filename.
+Supported: ❌
+Priority: medium
+
+`-O, --pcapfile PATH_TO_PCAPFILE` Write a PCAP of the ooniprobe session to this filename.
+Supported: ❌
+Priority: wontfix, we don't have packet capture support in probe-cli
+
+`-f, --configfile PATH_TO_CONFIG` Specify a path to the ooniprobe configuration file.
+Supported: ✅
+probe-cli equivalent: `--config`
+
+`-d, --datadir` Specify a path to the ooniprobe data directory.
+Supported: ✅
+probe-cli equivalent: set the `OONI_HOME` environment variable
+
+`-a, --annotations key:value[,key2:value2]` Annotate the report with a key:value[, key:value] format.
+Supported: ✅
+Priority: high
```

Comment:
Confusing: why add priority if supported? Also: you sure it's supported?

Reply:
Suggestion: `-a key:value -a key2:value2` is easier to handle than what we had previously. (MK uses `-A` for that precisely because the probe-legacy behaviour is not implemented.)

Reply:
(xref: https://github.com/ooni/probe-engine/issues/132)

Reply:
Ah yes you are right, this is not supported.